editor/src/components/menubar/menubar.tsx

```diff
@@ -15,51 +14,49 @@ import { shareURLForProject } from '../../core/shared/utils'
 import { isFeatureEnabled } from '../../utils/feature-switches'
 import {
   IcnProps,
-  SquareButton,
-  UtopiaStyles,
   FlexColumn,
   Tooltip,
   MenuIcons,
   LargerIcons,
   Avatar,
-  Icn,
-  Icons,
   UtopiaTheme,
 } from '../../uuiui'
 import { betterReactMemo } from '../../uuiui-deps'
-import { EditorAction, EditorDispatch } from '../editor/action-types'
+import { EditorAction } from '../editor/action-types'
 import { setLeftMenuTab, setPanelVisibility, togglePanel } from '../editor/actions/action-creators'
-import { EditorState } from '../editor/store/editor-state'
 import { useEditorState } from '../editor/store/store-hook'
 import { LeftMenuTab } from '../navigator/left-pane'
 
-const Tile = styled.div({
+interface TileProp {
+  size: keyof typeof UtopiaTheme.layout.rowHeight
+}
+
+const Tile = styled.div<TileProp>((props) => ({
   display: 'flex',
   flexDirection: 'column',
   justifyContent: 'center',
   alignItems: 'center',
-})
+  width: props.size,
+}))
 
 export interface MenuTileProps extends React.HTMLAttributes<HTMLDivElement> {
```

Comment:
Should this instead become `extends React.HTMLAttributes<HTMLDivElement>, TileProps`?